manifests/install.pp

```diff
@@ -0,0 +1,14 @@
+# **NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) CLASS**
```

Comment:
should be a bolt class, shouldn't it?

Reply:
No, this points to details on private classes.